contrib/init/dogecoind.opt.service

```diff
@@ -0,0 +1,74 @@
+# This variation of the unit file is for "opt" add-on installations that do not form part of the default installation.
+# (i.e. out of band installations by the user, not installed by a system package manager like "apt" via official sources)
```

Comment:
Nit: From the perspective of this repository and our release processes, "official" would be the release page here rather than the apt sources. So perhaps we can simply omit the last 3 words to not confuse people.